### DISCUSSION TITLE
Support for Session Cookies

### DISCUSSION THREAD

**Discussion by KgotsoK** [kreait/firebase-php#202] [Ideas]
#### 1. Which version of `kreait/firebase-php` are you using?
4.8.0

#### 2. Which version of PHP are you using?
7.1

#### 3. What's the issue?

Can we have support for Session Cookies:
`https://firebase.google.com/docs/auth/admin/manage-cookies`

It really simplifies Firebase Auth on stateful sites.

#### 4. Code that lead to the issue
N/A

**Comment by jeromegamez** [comment]
Please don‘t think that I‘m ignoring your feature request - I‘ve already looked at a possible implementation but am hesitating a little because working with cookies adds a fair bit of complexity and I‘m trying to do this right from the start. Stay tuned :)

**Comment by KgotsoK** [comment]
Thanks @jeromegamez . Glad to know you're playing with the idea. looking forward to it.

**Comment by jeromegamez** [comment]
@KgotsoK @TheMrg @Slapbox @marcus-at-localhorst I finally got around to start working on this in #244 - it would be great if you could try it out and tell me if and how that works for you \o/

**Comment by Nantris** [comment]
Unfortunately I'm not in a position to test currently, but this sounds like a great development!

**Reply by cosmini** [reply]
Any updates on this?
